buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -661,6 +662,25 @@ def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
 }
 
 def setup_pull_request() {
+    // Parse Github trigger comments
+    // For example:
+    /* This is great
+       Jenkins test extended <platform>
+       It was tested*/
+
+    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.tokenize(System.lineSeparator())
+    NUM_COMMENTS = PARSED_BY_NEWLINE_COMMENT.size()
+    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
+        def comment = COMMENT.tokenize(' ')
+        if (("${comment[0]}" == "Jenkins") && (("${comment[1]}" == "compile") || ("${comment[1]}" == "test"))) {
+            setup_pull_request_single_comment(comment)
+            return
+        }
+    }
+    error("Error: Invalid trigger comment")
```

Comment:
I was just posting the result of the PR build, adding the text "Error:" will cause the output to look like this
```
Error: Error Invalid trigger comment
```

Reply:
Please remove the extra "Error" string.